wagtail/embeds/embeds.py

```diff
@@ -42,6 +42,10 @@ def finder(url, max_width=None):
     if 'html' not in embed_dict or not embed_dict['html']:
```

Comment:
I guess its matter of taste but you can shortcut this (as well as `thumbnail_url` check/conversion) to smth like

```python
embed_dict['html'] = embed_dict.get('html') if isinstance(embed_dict.get('html'), str) else ''
embed_dict['thumbnail_url'] = embed_dict.get('thumbnail_url') if isinstance(embed_dict.get('thumbnail_url'), str) else ''
```

but again, thats just matter of taste. maybe your variant is more readable. 

you can always simply check if value is a string, because I guess we don't care if its _falsey_  (empty string, False, None, 0 etc) but rather if its a string type.

Reply:
Sorry I didn't see this before I pushed my (hopefully) last commit on this.